redis/asyncio/cluster.py

```diff
@@ -1572,9 +1572,9 @@ async def _execute(
 
             if raise_on_error:
                 for cmd in todo:
-                    name_exc = cmd.get_first_exception()
-                    if name_exc:
-                        name, exc = name_exc
+                    result = cmd.result
+                    if isinstance(result, Exception):
```

Comment:
I made this change this way so the diff against the original `redis-py` source is smaller.